source/assets/public/index.css

```diff
@@ -308,17 +311,67 @@ body {
     margin-top: 2rem;
   }
 
-.back-to-top {
-  text-decoration: none;
-  display: block;
-  width: 10rem;
-  margin-left: auto;
-  margin-right: auto;
+/* Sponsorship section */
+
+.c-sponsor-level__name {
+  font-family: var(--font-heading);
+  color: var(--primary-color);
+  font-size: 3rem;
+  margin-bottom: 2rem;
+  margin-top: 2rem;
```

Comment:
Same here. It's looking a little crammed in - especially at `breakpoint-tablet` and up.